src/CMakeLists.txt

```diff
@@ -1,16 +0,0 @@
-add_library(realm-js-shared OBJECT
```

Comment:
Why is this file deleted?

Reply:
That was my ide that did that by accident will delete it if not necessary :)